File: College-Sports-WebApp/Database/Models/ScoreboardResult.cs
namespace College_Sports_WebApp.Database.Models
{
    public class ScoreboardResult
    {
        public List<League> leagues { get; set; }
        public List<Video> videos { get; set; }
        public List<string> groups { get; set; }
        public List<Event> events { get; set; }
        public Zipcodes zipcodes { get; set; }
        public EventsDate eventsDate { get; set; }
    }

}

File: College-Sports-WebApp/ClientApp/src/app/models/scoreboard-result.ts
/* tslint:disable */
/* eslint-disable */
import { Event } from './event';
import { EventsDate } from './events-date';
import { League } from './league';
import { Video } from './video';
export interface ScoreboardResult {
  events?: Array<Event> | null;
  eventsDate?: EventsDate;
  groups?: Array<string> | null;
  id?: number;
  leagues?: Array<League> | null;
  videos?: Array<Video> | null;
}
